assets/sass/imports/_header.scss

```diff
@@ -251,3 +251,85 @@
     top: $container-padding;
   }
 }
+
+
+// Credit goes to https://www.w3schools.com/Css/css_dropdowns.asp for dropdown menu template, and to @aRoamingDuck#7049 for flex usage & re-organization help
+.menu-header {
+    display: flex;
+    flex-direction: row;
+}
+
+.menu-header .menu-item-header {
+    display: flex;
+    align-items: center;
+    font-size: 16px;
+    height: 45px;
+    background-color: inherit;
+    width: max-content;
+}
+
+.menu-header .dropdown  {
+    display: block;
+    font-size: 16px;
+    padding: 11px;
+    position: absolute;
+    top: 40px;
+    min-width: 140px;
+}
+
+.dropdown .dropdown-item {
+    display: none;
+    position: relative;
+    padding: 5px;
+    right: 10px;
+    vertical-align: middle;
+    align-items: center;
+    box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.5);
+}
+
+.dropdown-item {
+    display: none;
+    min-width: max-content;
+    height: 36px;
+    background-color: #fff;
+    margin: 0 auto;
+}
+
+.dropdown-item:hover {
+    background-color: #f1f1f1;
+}
+
+.menu-item-header:hover .dropdown .dropdown-item {
+    display: flex;
+}
+
+@media only screen and (max-width: 800px) {
+    .menu-header {
+        flex-direction: column;
+    }
+
+    .menu-header .menu-item-header {
+        display: flex;
+        flex-direction: column;
+        height: auto;
+        align-items:flex-start;
+        min-width: 100%;
+    }
+
+    .menu-header .dropdown {
+        position: relative;
+        top: 5px;
+        left: 30px;
+        padding: 0px 0px 12px 0px;
+    }
+
+    .dropdown .dropdown-item {
+        display: flex !important;
+        position: relative;
+        padding: 0px 1px !important;
+        box-shadow: 0px 0px 0px rgba(0,0,0,0)!important;
```

Comment:
Why are you using `!important` - it's highly discouraged to use it unless you've very good reason to?